README.md

```diff
@@ -98,6 +98,11 @@ Known edge cases and other topics are covered in [the FAQ](/FAQ.md).
 Contributions are welcome! Please read the [CONTRIBUTING.md](/CONTRIBUTING.md) document for details
 on getting started.
 
```

Comment:
Combine these two sections into one single section:
### Adding a Configuration Parameter to the SDK
###Steps to be followed progressively while proposing a new Configuration Parameter:
Replace both sections with:

## Adding a Configuration Parameter to the SDK
Configuration parameters allow users to flexibly leverage SDK features to match their performance, authentication, and encryption requirements.

### Submitting a new Configuration Parameter to the Java-Manta SKDK
- Determine the nature of the new parameter. Consider basic design decisions such as the conditions that govern its use, its datatype, etc.
- For example, if the new configuration parameter is a Boolean, find the line `Boolean isClientEncryptionEnabled();` in [ConfigContext](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/main/java/com/joyent/manta/config/ConfigContext.java). Use that section of code as a template to implement the new configuration variable.
- Add a method in  [SettableConfigContext](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/main/java/com/joyent/manta/config/SettableConfigContext.java) and match the pre-existing implementations in the class. Include additions to detailed information in the JavaDocs.
- Add keys for `constants` in [MapConfigContext](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/main/java/com/joyent/manta/config/MapConfigContext.java) and [EnvVarConfigContent](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/main/java/com/joyent/manta/config/EnvVarConfigContext.java). Include an implementation for the required interface methods.
- Add a default setting in [DefaultsConfigContext](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/main/java/com/joyent/manta/config/DefaultsConfigContext.java). This determines a default value for the parameter when none is provided via command line or environment variable.
- Add code to the methods `overwriteWithContext()` and `overwriteWithDefaultContext()` in [BaseChainedConfigContext](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/main/java/com/joyent/manta/config/SettableConfigContext.java) to ensure the overwrite behavior works correctly.
- Update the test file [ChainedConfigContextTest](https://github.com/joyent/java-manta/blob/master/java-manta-client-unshaded/src/test/java/com/joyent/manta/config/ChainedConfigContextTest.java) to verify the behavior for the added parameter.
- Update [USAGE.md](https://github.com/joyent/java-manta/blob/master/USAGE.md#configuration) to include the new configuration parameter to add clear instructions on how to use it.